include/wx/textctrl.h

```diff
@@ -270,6 +270,14 @@ enum wxTextAttrLineSpacing
     wxTEXT_ATTR_LINE_SPACING_TWICE          = 20
 };
 
+enum wxTextAttrUnderlineType
+ {
+     wxTEXT_ATTR_UNDERLINE_NONE              = 0,
+     wxTEXT_ATTR_UNDERLINE_SOLID             = 1,
+     wxTEXT_ATTR_UNDERLINE_DOUBLE            = 20,
```

Comment:
Why these particular values? Unless there is a good reason to use them, I'd remove all values for the enum elements entirely.

Reply:
OK, removed